plotly_resampler/figure_resampler/figure_resampler_interface.py

```diff
@@ -1336,7 +1336,7 @@ def _parse_dtype_orjson(series: np.ndarray) -> np.ndarray:
         #    * float16 and float128 aren't supported with latest orjson versions (3.8.1)
         #    * this method assumes that the it will not get a float128 series
         # -> this method can be removed if orjson supports float16
-        if series.dtype in [np.float16]:
+        if series.dtype in (np.float16):
```

Comment:
This line causes the test to fail, as this is not a tuple (& thus not iterable) but just a dtype with parentheses around it.